feature-libs/quote/components/header/overview/quote-header-overview.component.ts

```diff
@@ -0,0 +1,259 @@
+        quote.state === QuoteState.BUYER_OFFER)
+    );
+  }
+
+  /**
+   * Cancels the view of the edit card tile
+   * by setting the edit mode to 'false'.
+   */
+  cancel() {
+    this.editMode = false;
+  }
+
+  /**
+   * Saves the edited card tile.
+   *
+   * @param {Quote} quote - Quote
+   * @param {SaveEvent} event - edit event
+   */
+  save(quote: Quote, event: SaveEvent) {
+    this.editMode = false;
+    const metaData: QuoteMetadata = this.defineQuoteMetaData(event);
+
+    if (Object.getOwnPropertyNames(metaData).length >= 1) {
+      this.quoteFacade.editQuote(quote.code, metaData);
+    }
+  }
+
+  /**
+   * Toggles the edit mode.
+   */
+  toggleEditMode() {
+    this.editMode = !this.editMode;
+  }
+
+  /**
+   * Retrieves the card content that represents the quote information with its name and description.
+   *
+   * @param {string} name - Quote name
+   * @param {string} description - Quote description
+   * @returns {Observable<Card>} - Card content
+   */
+  getQuoteInformation(name?: string, description?: string): Observable<Card> {
+    return combineLatest([
+      this.translationService.translate('quote.header.overview.information'),
+      this.translationService.translate('quote.header.overview.name'),
+      this.translationService.translate('quote.header.overview.description'),
+    ]).pipe(
+      map(([infoTitle, nameTitle, descriptionTitle]) => {
+        return {
+          title: infoTitle,
+          paragraphs: [
+            {
+              title: nameTitle,
+              text: [name ?? QuoteHeaderOverviewComponent.NO_DATA],
+            },
+            {
+              title: descriptionTitle,
+              text: [description ?? QuoteHeaderOverviewComponent.NO_DATA],
+            },
+          ],
+        };
+      })
+    );
+  }
+
+  /**
+   * Retrieves the edit card content that represents the edit quote information with its name and description.
+   *
+   * @param {string} name - Quote name
+   * @param {string} description - Quote description
+   * @returns {Observable<EditCard>} - Edit card content
+   */
+  getEditQuoteInformation(name: string, description: string): EditCard {
+    return {
+      name: name,
+      description: description,
+      charactersLimit: QuoteHeaderOverviewComponent.CHARACTERS_LIMIT,
+    };
+  }
+
+  /**
+   * Retrieves the card content that represents the estimated total and expiry date information.
+   *
+   * @param {Quote} quote - Quote
+   * @param {any} expiryDate -  Expiry date
+   * @returns {Observable<Card>} - Card content
+   */
+  getEstimatedTotalAndExpiryDate(
+    quote: Quote,
+    expiryDate?: string | null
+  ): Observable<Card> {
+    const totalPrice =
+      this.getTotalPrice(quote) ?? this.getTotalPriceDescription(quote);
+    return combineLatest([
+      this.translationService.translate('quote.header.overview.priceAndExpiry'),
+      this.translationService.translate('quote.header.overview.estimatedTotal'),
```

Comment:
@Larisa-Staroverova Shouldn't we here use this.getTotalPriceDescription()?

Reply:
Done! [#18050](https://github.com/SAP/spartacus/pull/18050)